plugins/pixart-rf/pixart-rf.quirk

```diff
@@ -187,3 +187,9 @@ GType = FuPxiBleDevice
 [HIDRAW\VEN_093A&DEV_2852]
 Plugin = pixart_rf
 GType = FuPxiBleDevice
+
+# HP 430
+[HIDRAW\VEN_03F0&DEV_114C]
+Plugin = pixart_rf
+GType = FuPxiBleDevice
+Flags = is-hpac
```

Comment:
missing newline character at end of file

Reply:
solved in a3f8969ce1bb3ddf34e463aa232dc6861f6f6d3e